templates/preflight-job.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.secrets.create -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "psono.fullname" . }}-secret-creator
+  labels:
+    {{- include "psono.labels" . | nindent 4 }}
+  annotations:
+    helm.sh/hook: pre-install
+    helm.sh/hook-weight: "1"
+    helm.sh/hook-delete-policy: "hook-succeeded"
+spec:
+  template:
+    spec:
+      serviceAccountName: "{{ include "psono.fullname" . }}-sa"
+      automountServiceAccountToken: true
+      containers:
+        - name: secret-creator
+          image: harbor.cloudical.net/cci-tools/psono-helm-secret-helper:latest
```

Comment:
Please pin this version, might break the chart in the future otherwise

Reply:
finally had some time to fix this aswell :)